tools-local/init-build.proj

```diff
@@ -26,7 +26,7 @@
   </Target>
 
   <Target Name="BuildTasks">
-    <Exec Command="$(DotNetCliToolDir)dotnet build tasks\Microsoft.DotNet.SourceBuild.Tasks\Microsoft.DotNet.SourceBuild.Tasks.csproj" />
+    <Exec Command="$(DotNetCliToolDir)dotnet build tasks\Microsoft.DotNet.SourceBuild.Tasks\Microsoft.DotNet.SourceBuild.Tasks.csproj /p:SubmoduleDirectory=$(SubmoduleDirectory)" />
```

Comment:
Why did you need to pass it in explicitly?

Reply:
I was getting a weird failure after I updated it to include dir.props and so I thought maybe I was getting more than I wanted by including all of dir.props, so I switched it to just include the directory directly.  I'm going to switch it back and see if I can figure out the errors.